CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 1.0.0 - 2019-09-27
```

Comment:
Only one thing - can you put this under 'Unreleased'? Then I'll a separate pull request specifically to bump the version number.

Reply:
@injms Done 👍